logbus/solvermon/vertexmon.go

```diff
@@ -73,6 +74,10 @@ func (vm *vertexMonitor) parseError() {
 		internalStr = " internal"
 	}
 	switch {
+	case reHint.MatchString(errString):
```

Comment:
this will remove the Hint string from the error so it is only displayed in the help message at the end and not as part of the error message. This only applies to the new hint.Error and not the deprecated one